Add meta descriptions to Lisk Core docs, closes #371 (#393)
LiskHQ/lisk-docs: modules/ROOT/pages/migration.adoc

= Lisk Core Migration
Mona Bärenfänger <mona@lightcurve.io>

> :description: The Lisk Core Migration page describes how to migrate the node, if a new update introduces a hard fork in the network. It also provides background information about what exactly is happening during a migration.

Reviewer: ```suggestion
:description: The Lisk Core Migration page describes how to migrate the node, if a new update introduces a hard fork in the network. In addition, it also provides background information about all actions occurring during a migration.
```
